.eslintrc.json

```diff
@@ -64,6 +64,7 @@
 
   "rules": {
     "no-use-before-define": 0,
+    "no-plusplus": "off",
```

Comment:
Is this not the same as the template app?

Reply:
I think this might be enforced on the template. IMO this is the worst rule in the eslint ruleset, can revert it if really necessary though!